src/pages/home/report/ReportActionCompose.js

```diff
@@ -44,7 +44,8 @@ class ReportActionCompose extends React.Component {
         this.comment = props.comment;
         this.state = {
             isFocused: false,
-            textInputShouldClear: false
+            textInputShouldClear: false,
+            isCommentEmpty: this.props.comment.length > 0
```

Comment:
An "empty comment" would have a length of `0` so this logic is backwards.

Reply:
Please change this to 

`isCommentEmpty: props.comment.length === 0`

and then update the usages so the behavior does not change

Reply:
Done @marcaaron  Please check and let me know if i need to update any other changes.

Reply:
Thanks for the change. There is an unresolved comment from @shawnborton above.